wiki/en/Server-Troubleshooting.md

```diff
@@ -43,22 +43,14 @@ This should be your **public** IP address (find that with [Google](https://www.g
 
 You can set your server to listen on a custom port with the `--port` option. If you do this, you will need to tell people which port to connect on. They will need to append the port number to the address of your server in the format `[serverAddress]:[portNumber]`.
 
-### Is a private server a direct server without contact to the Jamulus Directory Server?
+### **_Headless Linux_**: I have created a server but it is joining a directory and I don't want it to.
```

Comment:
Not Linux specific.

Reply:
I think this is a pretty low-quality entry TBH and was tempted to delete it. It comes from a long list of similarly slightly odd things that were added to the wiki a while ago.

Reply:
Probably we can delete it since the default is now unregistered.